mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -2432,6 +2432,7 @@ def LLVM_CallIntrinsicOp
     OpBuilder<(ins "StringAttr":$intrin, "ValueRange":$args)>,
     OpBuilder<(ins "StringAttr":$intrin, "ValueRange":$args, "FastmathFlagsAttr":$fastMathFlags)>,
     OpBuilder<(ins "Type": $resultType, "StringAttr":$intrin, "ValueRange":$args)>,
+    OpBuilder<(ins "TypeRange": $resultTypes, "StringAttr":$intrin, "ValueRange":$args)>,
```

Comment:
Does the builder above that takes a single resultType rather than a TypeRange have many uses? If not it maybe worth to replace it with your new builder.